apps/web/src/app/[locale]/(dashboard)/[wsId]/structure.tsx

```diff
@@ -84,12 +83,42 @@ export function Structure({
     [pathname]
   );
 
+  // Helper function to handle time tracker navigation logic
+  const getTimeTrackerNavigation = useCallback(() => {
+    for (const section of links) {
+      if (section?.children) {
+        const timeTrackerLink = section.children.find((child) =>
+          child?.title?.toLowerCase().includes('tracker')
+        );
+
+        if (timeTrackerLink?.children) {
+          return {
+            currentLinks: timeTrackerLink.children,
+            history: [links, section.children],
+            titleHistory: [section.title, timeTrackerLink.title],
+            direction: 'forward' as const,
+          };
+        }
+      }
+    }
+    return null;
+  }, [links]);
+
```

Comment:
same here, the new update should be universal and usable for all features/products, not just time tracker alone